site/public/js/forum.js

```diff
@@ -1712,7 +1713,13 @@ function loadThreadHandler(){
 
                 $('#posts_list').empty().html(JSON.parse(json.data.html));
                 window.history.pushState({"pageTitle":document.title},"", url);
-
+                //Updates the title and breadcrumb
+                $(document).attr("title", thread_title);
+                if(thread_title.length > 25){
+                  $("h1.breadcrumb-heading").text(thread_title.slice(0,25)+"...");  
+                }else{
+                  $("h1.breadcrumb-heading").text(thread_title);
+                }
```

Comment:
This file isn't linted by `eslint`, but `eslint` generally enforces the suggested spacing rules.
```suggestion
                if (thread_title.length > 25) {
                  $('h1.breadcrumb-heading').text(thread_title.slice(0,25) + '...');  
                }
                else {
                  $('h1.breadcrumb-heading').text(thread_title);
                }
```